frontends/election-manager/src/components/smartcard_modal/program_super_admin_card_view.tsx

```diff
@@ -1,10 +1,20 @@
 import React from 'react';
-import { Button, Prose } from '@votingworks/ui';
+import styled from 'styled-components';
+import { Button, fontSizeTheme, HorizontalRule, Prose } from '@votingworks/ui';
 import { CardProgramming } from '@votingworks/types';
 
 import { generatePin } from './pins';
 import { SmartcardActionStatus, StatusMessage } from './status_message';
 
+interface HeadingProps {
+  marginTop: string;
+}
+
+const Heading = styled.h1<HeadingProps>`
+  /* stylelint-disable-next-line declaration-no-important */
+  margin-top: ${(props) => props.marginTop} !important;
+`;
+
```

Comment:
Would like to handle this in another way. I fear this is getting fragile and working against the spirit of the Prose component.

Reply:
As noted in https://github.com/votingworks/vxsuite/pull/2267#discussion_r937894753, agreed!